backend/services/container_migration_service.py

```
@@ -0,0 +1,260 @@
+                )
+            elif request.identifier_type == IdentifierType.NAME:
+                containers = await self.docker_ops.discover_containers_by_name(
+                    request.container_identifier, request.source_host, request.source_ssh_user
+                )
+            else:
+                containers = await self.docker_ops.discover_containers_by_labels(
+                    request.label_filters or {}, request.source_host, request.source_ssh_user
+                )
+
+            if not containers:
+                raise ValueError(f"No containers found matching criteria")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)